catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/databases/TableResourceTest.java

```diff
@@ -1446,7 +1446,7 @@ void patch_tableColumns_200_ok(TestInfo test) throws IOException {
     List<Column> columns = new ArrayList<>();
     columns.add(getColumn("c1", INT, USER_ADDRESS_TAG_LABEL).withDescription(null));
     columns.add(getColumn("c2", BIGINT, USER_ADDRESS_TAG_LABEL));
-    columns.add(getColumn("\"c.3\"", FLOAT, GLOSSARY1_TERM1_LABEL));
+    columns.add(getColumn("c.3", FLOAT, GLOSSARY1_TERM1_LABEL));
 
```

Comment:
Why are you changing this? This is a test for receiving quoted names in the API that we should unquote and store and still build the right FQN.